### PR TITLE
Reporting self-init is implemented

### PR DESCRIPTION
**Pull request by tsvrko**

### CODE REVIEW

**Review by nsidorevich — APPROVED** [review]

File: configuration/reporting-service/variables.env.original [hunk 1]
```diff
@@ -26,7 +42,7 @@ GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
 GITHUB_HOST=github.com
 
-DEBUG_ENABLED=false
+DEBUG_ENABLED=true
```
Comment (vdelendik) [inline configuration/reporting-service/variables.env.original:45]: let's restore false for debug by default

File: configuration/reporting-service/variables.env.original [hunk 2]
```diff
@@ -12,6 +12,22 @@ REDIS_PASSWORD=MdXVvJgDdz9Hnau7
 RABBITMQ_STOMP_HOST=rabbitmq
 RABBITMQ_STOMP_PORT=61613
 
+JENKINS_ENABLED=false
+JENKINS_URL=https://ci.qaprosoft.farm/
```
Comment (vdelendik) [inline configuration/reporting-service/variables.env.original:16]: default url should be:
http://localhost/jenkins

File: configuration/reporting-service/variables.env.original [hunk 3]
```diff
@@ -12,6 +12,22 @@ REDIS_PASSWORD=MdXVvJgDdz9Hnau7
...
+JENKINS_USER=admin
+JENKINS_PASSWORD=qaprosoft
```
Comment (vdelendik) [inline configuration/reporting-service/variables.env.original:18]: default password: changeit

File: configuration/reporting-service/variables.env.original [hunk 4]
```diff
@@ -12,6 +12,22 @@ REDIS_PASSWORD=MdXVvJgDdz9Hnau7
...
+JENKINS_FOLDER=stage
```
Comment (vdelendik) [inline configuration/reporting-service/variables.env.original:19]: default folder - empty value

**Pull request merged**